Packs/CiscoWSA/Integrations/CiscoWSAV2/CiscoWSAV2.py

```diff
@@ -0,0 +1,1663 @@
+
+    def access_policy_applications_update(
+        self,
+        policy_name: str,
+        application: str,
+        action: str,
+        values: dict[str, Any],
+        settings_status: str,
+    ) -> Response:
+        """
+        Update access policy's applications settings.
+
+        Args:
+            policy_name (str): Policy name to update.
+            application (str): Application to update.
+            action (str): Action to perform on values.
+            values (dict[str, Any]): Values to perform action on.
+            settings_status (str): Settings status for the policy.
+
+        Returns:
+            Response: API response from Cisco WSA.
+        """
+        data = {
+            "access_policies": [
+                {
+                    "policy_name": policy_name,
+                    "avc": {
+                        "applications": {
+                            application: {action: values}
+                            if action == "block"
+                            else {action: {value: {} for value in values}},
+                        },
+                        "state": settings_status,
+                    },
+                }
+            ]
+        }
+
+        return self._http_request(
+            "PUT",
+            f"{V3_PREFIX}/web_security/access_policies",
+            json_data=data,
+            resp_type="response",
+            ok_codes=[HTTPStatus.NO_CONTENT],
+        )
+
+    def access_policy_objects_update(
+        self,
+        policy_name: str,
+        object_type: str | None,
+        object_action: str | None,
+        object_values: List[str] | None,
+        block_custom_mime_types: List[str] | None,
+        http_or_https_max_object_size_mb: int | None,
+        ftp_max_object_size_mb: int | None,
+    ) -> Response:
+        """
+        Update access policy's objects settings.
+
+        Args:
+            policy_name (str): Policy name to update.
+            object_type (str | None): Object type.
+            object_action (str | None): Object action.
+            object_values (List[str] | None): Object values.
+            block_custom_mime_types (List[str] | None): Block custom MIME types.
+            http_or_https_max_object_size_mb (int | None): HTTP(S) max object size MB.
+            ftp_max_object_size_mb (int | None): FTP max object size MB.
+
+        Raises:
+            DemistoException: Policy was not found.
+            DemistoException: Update failed, objects were not found.
+
+        Returns:
+            Response: API response from Cisco WSA.
+        """
+        access_policies = self.access_policy_list(policy_name).get(
+            "access_policies", []
+        )
+        if not access_policies:
+            raise DemistoException("Policy was not found.")
+
+        if not access_policies[0].get("objects"):
+            raise DemistoException("Update failed, objects were not found.")
+
+        objects = access_policies[0].get("objects")
```

Comment:
This logic should handle in the command function and not in the Client request. ⚠️